readme.md

```diff
@@ -13,33 +13,30 @@ Utopia is an integrated design and development environment for React. It uses Re
 
 [Start the editor](https://utopia.app/project)
 
-# For contributors: Installing Utopia on your machine
+# For contributors only: Installing Utopia on your machine
 
-Utopia is browser-based. To run it locally, clone the repo, and then set up the server and webpack by following these instructions.
+(Please note: you shouldn't install Utopia on your machine in order to use it as a local dev environment: it's slower - sometimes very significantly so - and may still need connection to our servers. Also, you won't be able to edit projects in the file system on your machine if you install it locally.)
```

Comment:
I think we should remove the brackets here and make this bold to really emphasize the point

Reply:
I've reworded this slightly.